src/vicsetup/libvicsetup/hexdump.c

```diff
@@ -0,0 +1,161 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "hexdump.h"
+#include "raise.h"
+#include "malloc.h"
+
+static void _indent(size_t indent)
+{
+    for (size_t i = 0; i < indent; i++)
+        printf("  ");
+}
+
+void vic_hexdump_special(
```

Comment:
Special?

Reply:
Renamed to **vic_hexdump()** to **vic_hexdump_formatted()** and renamed **vic_hexdump_special()** to **vic_hexdump()**.